src/auth.ts

```diff
@@ -50,10 +50,14 @@ export async function leave({ withoutRedirect }: { withoutRedirect?: boolean } =
  *                    Pass `null` if working without permissions.
  * @param redirectTo Specify the URL you want users to return to.
  *                   Uses the current url by default.
+ * @param extraLobbyParams Extra parameters object. Properties on the
+ *                         object are converted to query params and sent
+ *                         to the auth lobby.
  */
 export async function redirectToLobby(
   permissions: Maybe<Permissions>,
-  redirectTo?: string
+  redirectTo?: string,
+  extraLobbyParams?: Record<string, string>
```

Comment:
Wondering if we might want to make this type more flexible? For example, would `Record<string, string | number>` make sense and we can convert `number`s to strings for the user? Maybe some other types?

Reply:
I personally prefer `Record<string,string>`, because that's what it'll be in the end in the URL. There's an infinite amount of types that *could* be converted to strings and no good way to choose which to support, I think.
Might be confusing if we choose to support converting `boolean` and `number`, but not `bigint` or `{ a: boolean, b: boolean }`.

Reply:
True, aside from `string`, it's hard to predict what other types would make sense and we'd just convert them to strings anyways. 

Let's keep it at `Record<string,string>` for now. If there is eventually a compelling reason to add more types, we can revisit this.